react/src/routes/youtubePlayer.jsx

```diff
@@ -0,0 +1,38 @@
+import { enableYouTubeTracking } from "@snowplow/browser-plugin-youtube-tracking";
+import React, { useEffect } from "react";
+
+export default function YoutubeTrack() {
+  const elemId = "youtube";
+
+  function loadVideoAndTrack() {
```

Comment:
We can define `loadVideoAndTrack` and `loadYouTubeIframeAPI` outside of the component function _(they would need to get the ID as input)_.